### ISSUE TITLE
Could not load type 'System.Security.Authentication.SslProtocols'

### ISSUE THREAD

**Issue by bluewalk** [Q42/Q42.HueApi#38]
Hi,

I've just started a project with getting the HueApi directly form NuGet and with just hitting run, I'm getting the following exception after about 15 seconds: 
`System.TypeLoadException: Could not load type 'System.Security.Authentication.SslProtocols' from assembly 'System.Net.Primitives, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.`

It looks like all the required packages have also been downloaded. Any clue?

[edit]
Seems that the library works on a Windows 7 machine, but not on my Windows 8.1 machine. Any ideas? Is there a reason the choice has been made for System.Net.Http?

Cheers

**Comment by michielpost** [comment]
What kind of project are you creating? Windows Store, Phone, website, WPF, etc?
Are you able to run the sample app included in this repo?

I'm also working on a Win8.1 machine and it should work without problems. Give me some more info about your setup and I'll try to reproduce the problem.

System.Net.Http includes the HttpClient which is used to make the http requests to the bridge.

**Comment by bluewalk** [comment]
Hi Michiel, I was trying to setup a poc using a simple winforms app, eventually this should run as a .net windows service. That failed so I downloaded the source and ran the test, there it actually showed the above error.
I'm using VS2013 Community, on the Win7 machine I've used VS2012 Pro, maybe that's the difference? So I assume this error sounds new to you? :) I'll give it another shot on another Win8.1 machine.
